cmd/snap/cmd_snap_op.go

```diff
@@ -270,6 +271,7 @@ func (mx modeMixin) asksForMode() bool {
 type cmdInstall struct {
 	channelMixin
 	modeMixin
+	Revision string `long:"revision" description:"Install the given revision (only for own snaps)"`
```

Comment:
the wording here is tricky; it's not "personal" snaps (especially not if I remember that that was used to describe something else entirely 6 months ago), because it can also be a snap you collaborate on. Any snap that appears in your myapps listing. In that sense "own" isn't much better. How about

    "Install the given revision. You must be logged in, and can only specify non-default revisions for snaps to which you have developer access"

?

That's close to completely correct (completely correct in my mind would include the fact that specifying it for a revision you don't have access to, meaning a revision that isn't in the channel map if you are not a developer of the snap, or is not published if you are, will result in a "not found" error). But it is a little long for a help string, I think. Maybe shortening it to

    "Install the given revision of a snap to which you must have developer access"

would be acceptable?